skimage/registration/_tvl1.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+"""TV-L1 optical flow algorithm implementation.
+
+"""
+
+from functools import partial
+import numpy as np
+from scipy import ndimage as ndi
+from skimage.transform import warp
+
+from .utils import coarse_to_fine
+
+
+def _tvl1(I0, I1, flow0, dt, lambda_, tau, nwarp, niter, tol, prefilter):
```

Comment:
I wonder if it'd be possible to have more "explanatory" variable names here and throughout the code. For instance, if `I0` is the first grayscale image, couldn't we use something as `first_image`, or `first_frame` (as in the paper? This way, we would have `second_image`, `vector_init`, `step`, `attach_par`, and `tight_par. It'd be very valuable for someone who doesn't know the algorithm (myself included) :)

Reply:
imho we should go for `image0` and `image1` or `reference` and `target`/`moving` to be consistent with other registration methods.

Also, no need to add a `_par` suffix to parameters! `attachment`, `tightness` is sufficient.